### PR TITLE
added live demo to readme and changed project title in index.html

### PR DESCRIPTION
**Pull request by m05e5**
CHANGES APPLIED 

- I added a link to my live demo in the readme.md
- I changed the page title in the index.html file

### CODE REVIEW

**Review by Temmarie — COMMENTED** [review]
## Status: Invalid ❗ 

### Hi @m05e5  👋🏾 , 

 **While you made a great effort in this project, unfortunately, I cannot proceed to review your code.**

### Invalid Code Review Request

I have to mark this submission as invalid because you skipped the following: 

- [ ]  You have to write a descriptive Pull Request Message with a descriptive title as well. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.

#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

#### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by m05e5** [comment]
thanks for your review. I think now it is better

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Here is my responsive portfolio
 - Html and CSS
 - VS studio as IDE
 
+## Live Demo
+
+[Live Demo Link](https://m05e5.github.io/moses_portfolio)
+
 
 ### Prerequisites
```
Comment (Temmarie) [inline README.md]: - [ ] A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). 
- Focus on the instructions you gave for cloning the repo and running it locally.
- The screenshot you added in your readme is broken. Kindly fix it.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @m05e5     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Recommended** ♻️
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: README.md (hunk 1 above)
Comment (m05e5) [reply README.md]: Ok thanks for the review I will apply the changes but you didn't mention that in the first review

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉 🎉 

Hi @m05e5, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Temmarie) [reply README.md]: Hi @m05e5 , I didn't mention that in the first review because it was invalidated. 😇

**Comment by m05e5** [comment]
thanks

**Pull request merged**